astropy/utils/introspection.py

```diff
@@ -111,6 +121,19 @@ def minversion(module, version, inclusive=True, version_path='__version__'):
     True
     """
 
+    if isinstance(module, types.ModuleType):
+        module_name = module.__name__
+    elif isinstance(module, six.string_types):
+        module_name = module
+        try:
+            module = resolve_name(module_name)
+        except ImportError:
+            return False
```

Comment:
There are some cases where even this isn't quite right.  For example, say there's a bug in Numpy 1.8.x (all versions) but it's fixed in Numpy 1.9.0.dev1234567.  Then what we *really* want, to check for that bug, is specifically `not minversion(numpy, '1.9.0.dev1234567')` :(  So even still this is just a first approximation.